lib/rdkafka/bindings.rb

```diff
@@ -161,6 +165,15 @@ class TopicPartitionList < FFI::Struct
       end
     end
 
+    OAuthbearerTokenRefreshCallback = FFI::Function.new(
```

Comment:
The tests serve as a form of documentation.  I see there is https://github.com/karafka/wiki.  Do I need to add anything to that about intended usage of the callback?  Specifically around the global nature of callback and the intent of sending back the client name.  A comment on top of this callback?

Reply:
You can add the cooment on the callback and it will auto-propagate to the docs